src/python_bindings/examples/mine_typos.py

```diff
@@ -4,23 +4,41 @@
 
 from colorama import Style, Fore
 from jellyfish import levenshtein_distance
-import desbordante as desb
-import pandas as pd
+import desbordante
+import pandas
 
+# Value cluster filtering parameters.
 RADIUS = 3
 RATIO = 0.1
-ERROR = 0.005
-EXACT_ALGORITHM = 'FastFDs'
-EXACT_CONFIG = {}
-APPROXIMATE_ALGORITHM = 'Pyro'
-APPROXIMATE_CONFIG = {'error': ERROR}
-DATASET_PATH = 'Workshop.csv'
+
+# Algorithm that finds exact FDs and its config.
+EXACT_ALGORITHM_TYPE = desbordante.HyFD
+EXACT_ALGO_CONFIG = {}
+
+# Algorithm that finds approximate FDs and its config.
+APPROXIMATE_ALGORITHM_TYPE = desbordante.Pyro
+ERROR = 0.005  # Highest error for almost holding FDs.
+APPROXIMATE_ALGO_CONFIG = {'error': ERROR}
+
+# Parameters for pandas.read_csv(...).
+DATASET_PATH = '../../../test_input_data/Workshop.csv'
```

Comment:
I don't think it's a good idea to hardcode relative paths (or at least paths with `..`) in the code